hw/ip/kmac/pre_syn/syn_setup.example.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Setup IP name and top module.
+# When changing the top module, some parameters might not exist and
+# tcl/yosys_run_synth.tcl might need to be adjusted accordingly.
+export LR_SYNTH_IP_NAME=kmac
+export LR_SYNTH_TOP_MODULE=keccak_2share
+
+# Setup module parameters. The Width is actually 1600 but it can have any value
+# in [25, 50, 100, 200, 400, 800, 1600]. We choose 25 instead to get smaller
+# netlist thereby speeding up the formal masking verification.
+export LR_SYNTH_WIDTH=25
+export LR_SYNTH_EN_MASKING=1
+
+# Setup cell library path.
+# Uncomment the lines below and set the path to an appropriate .lib file
+# export LR_SYNTH_CELL_LIBRARY_PATH=/path/to/NangateOpenCellLibrary_typical.lib
+# export LR_SYNTH_CELL_LIBRARY_NAME=nangate
```

Comment:
This confused me at first, as I interpreted PATH as the path to the nangate-lib folder and NAME as the actual lib.
On a second look, I realized that the `/path/to/NangateOpenCellLibrary_typical.lib` includes the `.lib`
Perhaps we could uncomment line 22 by default, as there is no other choice for the variable atm?